### PR TITLE
Youtube parse

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "draft-js": "^0.10.4",
     "firebase": "^4.8.1",
     "globie-styl-library": "^1.0.0",
+    "js-video-url-parser": "^0.2.6",
```
Comment (varffs) [inline package.json:11]: Do we have a policy on what packages we can use. This is a low version number with low user count. But does that matter?

File: src/components/noticias/NoticiaForm.jsx [hunk 2]
```diff
@@ -259,7 +292,21 @@ class NoticiaForm extends Component {
           </div>
         </div>
 
-        <Uploads
+        <div className='grid-row margin-bottom-basic'>
```
Comment (varffs) [inline src/components/noticias/NoticiaForm.jsx:295]: That we have repeated code here looks wrong. Should this not be a custom form component that we use across contexts?

File: src/components/artistas/ArtistaForm.jsx [hunk 3]
```diff
@@ -231,6 +264,20 @@ class ArtistaForm extends Component {
...
+        <div className='grid-row margin-bottom-basic'>
+          <div className='grid-item item-s-12'>
+            <h4 className='font-size-small font-bold margin-bottom-tiny'><label htmlFor='video'>Video</label></h4>
+            <input
+              id='video'
+              name='video'
+              type='text'
+              disabled={this.state.isLoading}
+              value={this.state.video.url}
+              onChange={ event => this.handleVideoChange(event.target.value)}
```
Comment (varffs) [inline src/components/artistas/ArtistaForm.jsx:276]: The issue with this event is that you cannot type you have to paste in the content. Perhaps this is fine (who is going to manually copy the url? But it does mean that the input should inform the user they have to paste. Or perhaps we should just debounce the trigger?

File: src/components/artistas/ArtistaForm.jsx [hunk 4]
```diff
@@ -116,6 +122,33 @@ class ArtistaForm extends Component {
...
+          id,
+          url,
+          provider,
+        }
+      });
+    } else {
+      this.setState({
+        video: {
+          url,
+          url: '',
```
Comment (mikeyrayvon) [inline src/components/artistas/ArtistaForm.jsx]: seems like a duplicate

File: src/components/noticias/NoticiaForm.jsx [hunk 5]
```diff
@@ -172,6 +179,32 @@ class NoticiaForm extends Component {
...
+          id,
+          url,
+          provider,
+        }
+      });
+    } else {
+      this.setState({
+        video: {
+          url,
+          url: '',
```
Comment (mikeyrayvon) [inline src/components/noticias/NoticiaForm.jsx]: here as well

**Review by varffs — COMMENTED** [review]
Shit I forgot to hit go on my review damn

File: src/components/artistas/ArtistaForm.jsx (hunk 3 above)
Comment (mixcas) [reply src/components/artistas/ArtistaForm.jsx:276]: I think debounce is the way to go. I was going to do it but I forgot, sorry.

**Pull request merged**